templates/service-tls-template.yml

```diff
@@ -0,0 +1,553 @@
+            image: ${IMAGE_REGISTRY}/${IMAGE_REPOSITORY}:${IMAGE_TAG}
+            imagePullPolicy: IfNotPresent
+            volumeMounts:
+            - name: tls
+              mountPath: /secrets/tls
+            - name: grpc-server-tls
+              mountPath: /secrets/grpc-server-tls
+            - name: grpc-broker-tls
+              mountPath: /secrets/grpc-broker-tls
+            - name: service
+              mountPath: /secrets/service
+            - name: rds
+              mountPath: /secrets/rds
+            - name: ${MESSAGE_DRIVER_TYPE}
+              mountPath: /secrets/${MESSAGE_DRIVER_TYPE}
+            - name: authentication
+              mountPath: /configs/authentication
+            - name: mqtt-certs
+              mountPath: /secrets/mqtt-certs
+            - name: maestro-grpc-cert
+              mountPath: /secrets/maestro-grpc-cert
+            env:
+              - name: "AMS_ENV"
+                value: "${ENVIRONMENT}"
+              - name: POD_NAME
+                valueFrom:
+                  fieldRef:
+                    fieldPath: metadata.name
+            command:
+            - /usr/local/bin/maestro
+            - server
+            - --subscription-type=${SUBSCRIPTION_TYPE}
+            - --client-id=maestro-$(POD_NAME)
+            - --db-host-file=/secrets/rds/db.host
+            - --db-port-file=/secrets/rds/db.port
+            - --db-user-file=/secrets/rds/db.user
+            - --db-password-file=/secrets/rds/db.password
+            - --db-name-file=/secrets/rds/db.name
+            - --db-rootcert=/secrets/rds/db.ca_cert
+            - --db-sslmode=${DB_SSLMODE}
+            - --message-broker-type=${MESSAGE_DRIVER_TYPE}
+            - --message-broker-config-file=/secrets/${MESSAGE_DRIVER_TYPE}/config.yaml
+            - --enable-ocm-mock=${ENABLE_OCM_MOCK}
+            - --ocm-client-id-file=/secrets/service/ocm-service.clientId
+            - --ocm-client-secret-file=/secrets/service/ocm-service.clientSecret
+            - --ocm-base-url=${OCM_BASE_URL}
+            - --ocm-debug=${OCM_DEBUG}
+            - --https-cert-file=/secrets/tls/tls.crt
+            - --https-key-file=/secrets/tls/tls.key
+            - --enable-grpc-server=${ENABLE_GRPC_SERVER}
+            - --disable-grpc-tls=${DISABLE_GRPC_TLS}
+            - --grpc-tls-cert-file=/secrets/grpc-server-tls/tls.crt
+            - --grpc-tls-key-file=/secrets/grpc-server-tls/tls.key
+            - --grpc-broker-tls-cert-file=/secrets/grpc-broker-tls/tls.crt
+            - --grpc-broker-tls-key-file=/secrets/grpc-broker-tls/tls.key
+            - --grpc-server-bindport=${GRPC_SERVER_BINDPORT}
+            - --grpc-broker-bindport=${GRPC_BROKER_BINDPORT}
+            - --grpc-client-ca-file=/secrets/maestro-grpc-cert/ca.crt
+            - --grpc-authn-type=token
+            - --grpc-tls-cert-file=/secrets/maestro-grpc-cert/server.crt
+            - --grpc-tls-key-file=/secrets/maestro-grpc-cert/server.key
```

Comment:
we use this self signed certificate for auth.

Reply:
remove the `grpc-server-tls` and only use our self signed certificates